app/assets/src/components/common/Header.jsx

```diff
@@ -149,21 +149,28 @@ const UserMenuDropDown = ({
 
   const renderItems = adminUser => {
     let userDropdownItems = [];
-    adminUser &&
+
+    allowedFeatures.includes("bulk_downloads") &&
       userDropdownItems.push(
         <BareDropdown.Item
-          key="3"
+          key="1"
           text={
-            <a className={cs.option} href="/users/new">
-              Create User
+            <a
+              className={cs.option}
+              href="/bulk_downloads"
+              onClick={() =>
+                logAnalyticsEvent("Header_dropdown-dropdowns-option_clicked")
```

Comment:
I see this was moved code, but you probably meant `downloads-option_clicked`

Reply:
Great catch!!